### ISSUE TITLE
Testing + inference code

### ISSUE THREAD

**Issue by isaacmg** [AIStream-Peelout/flow-forecast#34]
- [x] Create code to forecast N timestamps into the future 
- [x] Optional parameter to specify real values
- [x] Keep track of timestamps and return df of time steps plus forecast
- [x] Take list of metrics to compute as param to enable several metrics.
- [x] Log to Wandb
- [x] Add run on test set at end of training loop.
